src/components/bs5/quickexit/quickexit.scss

```diff
@@ -149,21 +150,30 @@
     position: sticky;
     inset-block-start: 0;
     z-index: 999;
-    padding-block: 20px; //
-    padding-inline: 40px; //
+    padding-block: 20px;
+    padding-inline: 40px;
+
+    .container {
+      padding-inline: calc(var(--#{$prefix}gutter-x) * 0.5);
+    }
 
     &-inner {
       justify-content: space-between;
 
       .qld-quick-exit-tip-link {
         -ms-flex-align: center;
```

Comment:
Hey @SenChung - this SCSS will be compiled through a PostCSS plugin in our build script, which adds most vendor prefixes automatically.  Might save you a few extra lines of code ;)

Reply:
the -ms-flex-align was added before I touched this file.
I left this, as whoever added must have needed for a very specific reason. 
That said Thanks @stvp-qld for reaching out to me on this, as these types of things can eat-up loads of time.